src/server/server.rs

```diff
@@ -199,6 +203,7 @@ impl<T: RaftStoreRouter> Server<T> {
             }
             MessageType::Cmd => self.on_raft_command(msg.take_cmd_req(), token, msg_id),
             MessageType::KvReq => self.store.on_request(msg.take_kv_req(), token, msg_id),
+            MessageType::CopReq => self.coprocessor.on_request(msg.take_cop_req(), token, msg_id),
```

Comment:
Yeah. We will add thread inside coprocessor (just like Storage) later, this PR concentrates on the snapshot.